### PR TITLE
Update picture-entity.markdown

### PR DESCRIPTION
**Pull request by robehem**
## Proposed change
Added link to documentation about hosting files.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ x] The documentation follows the Home Assistant documentation [standards][].

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @robehem, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16285).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_lovelace/picture-entity.markdown [hunk 1]
```diff
@@ -31,7 +31,7 @@ camera_view:
   type: string
 image:
   required: false
-  description: URL of an image.
+  description: URL of an image.<br/>To use a locally hosted image, see [Hosting](/integrations/http#hosting-files).
```
Comment (frenck) [inline source/_lovelace/picture-entity.markdown]: We should not use HTML here.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_lovelace/picture-entity.markdown (hunk 1 above)
Comment (robehem) [reply source/_lovelace/picture-entity.markdown]:
```suggestion
  description: URL of an image. To use a locally hosted image, see [Hosting](/integrations/http#hosting-files).
```

Comment (robehem) [reply source/_lovelace/picture-entity.markdown]: Oh I see! I copied it from here and thought it was correct:
https://www.home-assistant.io/lovelace/picture-elements/

**Review by frenck — APPROVED** [review]
Thanks, @robehem 👍

**Pull request merged**